source/lambda/apiprocessor/lambda_function.py

```diff
@@ -62,6 +81,13 @@ def lambda_handler(event, context):
                     result = deleteDocument(request)
                     deleteESItem(request["elasticsearchDomain"], request["documentId"])
 
+                    # remove it from Kendra's index too if present
+                    if 'KENDRA_INDEX_ID' in os.environ :
+                        kendraClient = KendraHelper()
+                        kendraClient.deindexDocument(os.environ['KENDRA_REGION'],
```

Comment:
You can get the region from the Lambda pre-set environment variables as mentioned above